app/src/main/java/pl/gittobefit/database/entity/UserEntity.java

```diff
@@ -10,47 +10,66 @@
  * @author Czapla
  */
 @Entity
-public class UserEntity {
+public class UserEntity
+{
     @PrimaryKey()
     private int id;
 
     private String email;
     private String token;
+    private boolean google;
```

Comment:
```suggestion
    private boolean isLoggedByGoogle;
```
Poza tym, czy do logowania przez fb nie musi być coś podobnego zrobionego?